packages/cactus-plugin-bungee-hermes/src/test/typescript/integration/fabric-test-pruning.test.ts

```diff
@@ -333,99 +333,122 @@ beforeAll(async () => {
       instanceId: uuidv4(),
     };
 
-    networkDetails = {
-      connectorApiPath: fabricPath,
-      signingCredential: fabricSigningCredential,
-      channelName: fabricChannelName,
-      contractName: fabricContractName,
-      participant: "Org1MSP",
-    };
+    networkDetailsList = [
```

Comment:
please add one standalone test that verifies that the connectors with which BUNGEE were initialized work (one test file for all connectors; 1 test case for each connector). you may use the simplest connector calls